api/graylog.go

```diff
@@ -209,8 +209,11 @@ func UpdateRegistration(httpClient *http.Client, checksum string, ctx *context.C
 	respBody := new(graylog.ResponseCollectorRegistration)
 	resp, err := c.Do(r, &respBody)
 	if resp != nil && resp.StatusCode == 400 && strings.Contains(err.Error(), "Unable to map property") {
```

Comment:
This comment is not about your change but the behavior of the sidecar here. Do you know why we disable the status if we detect the substring in the error message?

If the substring would indicate "send_status" not working, then the log wouldn't show up more than once because we disabled the status submission. Or do I miss anything?

I also don't understand the "Disabling `send_status` as fallback." message. What does fallback mean here?

Reply:
I'm guessing this was put in place to deal with older servers that didn't support status requests.
It catches the generic jackson error about unknown properties.

The idea of the "fallback" was to register at least a minimal request from the sidecar.
otherwise it would not be registered at all.

I've also made a change to the server that relaxes the NodeDetails with `@JsonIgnoreProperties(ignoreUnknown = true)`